handlers/controller.js

```diff
@@ -149,29 +113,12 @@ let addProductByLookup = async(req, res) => {
         // else
         // query barcodelookup for product
         let bclRes = await bc.lookup({key: process.env.BC_API_KEY, barcode: barcode});
+        console.log(bclRes);
         if(bclRes.statusCode !== StatusCode.OK) {
             console.error(`error looking up ${barcode} in barcodelookup`);
             return res.status(bclRes.statusCode).send({ data: bclRes.data });
         }
-
-        let newProduct = new models.Product ({
```

Comment:
Are we not doing this product structure anymore and straightaway adding data? I don't see the below default initializations anywhere.

Reply:
Look at the newly added `Controller` file inside of `utils` @ShekharDewan.